### PR TITLE
PreProcessingChannel API

### LINKED ISSUE #26: Support PreProcessChannel in PreProcessInfo interface
It is required to support mean and std values. Please refer to the API doc of [PreProcessChannel](https://docs.openvinotoolkit.org/latest/structInferenceEngine_1_1PreProcessChannel.html#abde989b919c44225ea30dbacb64f34b2).

### PR DESCRIPTION
**Pull request by artyomtugaryov**

### CODE REVIEW

**Review by newblaque — APPROVED** [review]

**Comment by huningxin** [comment]
@artyomtugaryov , I am wondering whether this PR is ready for review. I see it has WIP label.

**Comment by artyomtugaryov** [comment]
I think you can review this pool request, but this API does not affect inputs as API that we have in the master branch. I just rework PreProcessingChannel API

File: test/network.js [hunk 1]
```diff
@@ -331,29 +331,29 @@ describe('Network Test', function() {
...
-  it('PreProcessChannel should throw when the number of channels is 0', () => {
-    const preprocessInfo = network.getInputsInfo()[0].getPreProcess();
-    expect(preprocessInfo.getNumberOfChannels()).to.be.a('number').equal(0);
-    expect(() => preprocessInfo.getPreProcessChannel(0)).to.throw();
-  });
+  it('PreProcessChannel should throw when the number of channels is 0',
+     () => {
+         // const preprocessInfo = network.getInputsInfo()[0].getPreProcess();
+         // expect(preprocessInfo.getNumberOfChannels()).to.be.a('number').equal(0);
+         // expect(() => preprocessInfo.getPreProcessChannel(0)).to.throw();
```
Comment (huningxin) [inline test/network.js]: If we don't need this test case any more, it's better to remove it.

File: src/preprocess_channel.cc [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+                                    const Napi::Value& value) {
+  auto stdScale = value.As<Napi::Number>();
+  this->_actual->stdScale = stdScale.FloatValue();
+}
+
+Napi::Object PreProcessChannel::NewInstanceAsync(
+    const Napi::Env& env,
+    const InferenceEngine::PreProcessInfo& preProcessInfo,
+    const size_t& index,
+    Napi::Promise::Deferred& deferred) {
```
Comment (huningxin) [inline src/preprocess_channel.cc]: I suppose 'PreProcessChannel' can be created synchronously, does it? If it is, we don't need the `NewInstanceAsync`.

**Review by huningxin — COMMENTED** [review]
Thanks @artyomtugaryov ! I left some comments, please take a look. And please update the api_doc.md as well.

**Comment by artyomtugaryov** [comment]
@huningxin @lionkunonly can you review this PR?

**Review by huningxin — APPROVED** [review]
Looks good, thanks @artyomtugaryov !

**Comment by artyomtugaryov** [comment]
@huningxin can you merge it?

**Comment by huningxin** [comment]
Sure, I am happy to merge it, thanks again.

**Pull request merged**